inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java

```diff
@@ -160,7 +160,9 @@ public Map<String, String> tableOptions() {
 
         options.put(HUDI_OPTION_DATABASE_NAME, dbName);
         options.put(HUDI_OPTION_TABLE_NAME, tableName);
-        options.put(HUDI_OPTION_RECORD_KEY_FIELD_NAME, primaryKey);
+        if (StringUtils.isNotEmpty(primaryKey)) {
```

Comment:
Using `isNotEmpty` without considering the situation of '', using `isNotBlank` may be better.

Reply:
fixed thanks